src/modules/live-execution-tracker/services/live-execution-tracker.service.ts

```diff
@@ -636,38 +636,25 @@ export class LiveExecutionTrackerService implements ILiveExecutionTrackerService
   }
 
   private async importXmlIntoDiagramModeler(diagramModeler: IBpmnModeler, xml: string): Promise<void> {
-    const xmlImportPromise: Promise<void> = new Promise((resolve: Function, reject: Function): void => {
-      diagramModeler.importXML(xml, (importXmlError: Error) => {
-        if (importXmlError) {
-          reject(importXmlError);
-
-          return;
-        }
-        resolve();
-      });
-    });
-
-    return xmlImportPromise;
+    try {
+      await diagramModeler.importXML(xml);
+      return Promise.resolve();
+    } catch (error) {
+      return Promise.reject(error);
+    }
```

Comment:
```suggestion
    await diagramModeler.importXML(xml);
```

Auch hier macht die Funktion keinen Sinn mehr